etcdctl/ctlv3/command/printer.go

```diff
@@ -238,6 +238,8 @@ func makeEndpointStatusTable(statusList []epStatus) (hdr []string, rows [][]stri
 			fmt.Sprint(status.Resp.RaftIndex),
 			fmt.Sprint(status.Resp.RaftAppliedIndex),
 			fmt.Sprint(strings.Join(status.Resp.Errors, ", ")),
+			fmt.Sprint(status.Resp.DowngradeInfo.GetTargetVersion()),
+			fmt.Sprintf("%v", status.Resp.DowngradeInfo.GetEnabled()),
```

Comment:
I know this probably follows the previous lines. But I don't think we need to do a `fmt.Sprintf(...)` on the string values. Or is there a reason to do that? (Sorry, I'm not very familiar with `etcdctl`).

I mean:

```suggestion
			status.Resp.DowngradeInfo.GetTargetVersion(),
			strconv.FormatBool(status.Resp.DowngradeInfo.GetEnabled()),
```

(The bool conversion raises a [`perfsprint`](https://github.com/catenacyber/perfsprint) linter warning).

Reply:
I agree that `strconv.FormatBool` is better, because obvious it has a little performance. I think it's OK to update it separately,
- It isn't critical
- To keep consistent with other bool (i.e. `fmt.Sprint(status.Resp.IsLearner)`) , you can just use `fmt.Sprint(status.Resp.DowngradeInfo.GetEnabled())`.

Reply:
@ivanvc thanks! Updated